test/e2e/setup/test_config.go

```diff
@@ -0,0 +1,31 @@
+package setup
+
+import (
+	"github.com/mongodb/mongodb-kubernetes-operator/pkg/util/envvar"
+)
+
+const (
+	testNamespaceEnvName      = "TEST_NAMESPACE"
+	operatorImageEnvName      = "OPERATOR_IMAGE"
+	clusterWideEnvName        = "CLUSTER_WIDE"
+	versionUpgradeHookEnvName = "VERSION_UPGRADE_HOOK_IMAGE"
+	performCleanupEnvName     = "PERFORM_CLEANUP"
+)
+
+type testConfig struct {
+	namespace               string
+	operatorImage           string
+	versionUpgradeHookImage string
+	clusterWide             bool
+	performCleanup          bool
+}
+
+func loadTestConfigFromEnv() testConfig {
+	return testConfig{
+		namespace:               envvar.GetEnvOrDefault(testNamespaceEnvName, "default"),
+		operatorImage:           envvar.GetEnvOrDefault(operatorImageEnvName, "quay.io/mongodb/community-operator-dev:latest"),
+		versionUpgradeHookImage: envvar.GetEnvOrDefault(versionUpgradeHookEnvName, "quay.io/mongodb/mongodb-kubernetes-operator-version-upgrade-post-start-hook:1.0.2"),
```

Comment:
I see that there is a default value here for the `versionUpgradeHookImage`- why is it hard-coded in e2e.py:157?

Reply:
I will add a separate smaller PR to configure this (reason mentioned in the other comment!)